theme/static/style/color-palette-light.css

```diff
@@ -63,3 +63,16 @@ nav.primary a.active, #pagination a.active {
     background-color: #6e827b;
     color: #e3edea;
 }
+
+#mc_embed_signup {
+    background-color: #ededed;
```

Comment:
I think you have to define text color for the edit field b/c when I type I don't see what I've typed into this field. Everything looks white.

Reply:
The problem was in the combination of `box-sizing: border-box` and `padding: 2.5rem 0` for the text box's styles.

For some reason this is only present in Firefox.